src/utils/arrangerUtils.ts

```diff
@@ -0,0 +1,205 @@
+import flatten from 'lodash/flatten';
+import isPlainObject from 'lodash/isPlainObject';
+import isEmpty from 'lodash/isEmpty';
+import { Client } from '@elastic/elasticsearch';
+
+type CacheElem = {
+    key: string;
+    entry: any;
+    expiry: number;
+};
+
+type Entries = {
+    [key: string]: CacheElem;
+};
+
+/**
+ * An in-memory, asynchronous cache.
+ */
+class AsyncCache {
+    /**
+     * Time to live of an item in the cache
+     */
+    ttl = 24 * 60 * 60 * 1000;
```

Comment:
not your code, but maybe : `const ONE_SEC_IN_M_SEC = 24 * 60 * 60 * 1000` or something like that.